csrc/mmdeploy/net/torchscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+project(mmdeploy_torch_net)
+
+find_package(Torch REQUIRED)
+find_package(TorchVision)
```

Comment:
In what case `torchvision` is needed?

Reply:
when model contains operators from torchvision, like `torchvision.ops.nms`.